app/javascript/react/components/SessionsListView/SessionsListView.style.tsx

```diff
@@ -1,18 +1,39 @@
 import styled from "styled-components";
 
-import { gray100 } from "../../assets/styles/colors";
+import { blue, gray100, gray400 } from "../../assets/styles/colors";
+import { SessionListTile } from "./SessionsListTile/SessionListTile.style";
 
 const SessionListViewStyle = styled.div`
   position: fixed;
   top: 10.5rem;
   bottom: 0;
```

Comment:
The last session on the list is always partially covered by the threshold, maybe we can just shorten it setting the bottom just right above the threshold on desktop view while we're at it? It's not related to the main PR, but since we touch this part it might be a good idea to get this dealt with.

Reply:
I will fix it :)